src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,45 @@
+package chess.domain.board;
+
+import chess.domain.piece.Piece;
+import chess.domain.piece.state.PiecesState;
+import chess.domain.piece.state.Result;
+import chess.domain.position.MovingFlow;
+import chess.domain.position.Position;
+
+import java.util.Map;
+
+public class Board {
+    private final PiecesState piecesState;
+
+    private Board(PiecesState piecesState) {
+        this.piecesState = piecesState;
+    }
+
+    public static Board initialize() {
+        PiecesState piecesState = PiecesState.initialize();
+        return new Board(piecesState);
+    }
+
+    public Board movePiece(MovingFlow movingFlow) {
+        Position from = movingFlow.getFrom();
+        Position to = movingFlow.getTo();
+        PiecesState piecesState = this.piecesState.movePiece(from, to);
+        return new Board(piecesState);
+    }
+
+    public boolean isNotFinished() {
+        return piecesState.isNotFinished();
+    }
+
+    public Result concludeResult() {
+        return piecesState.concludeResult();
```

Comment:
```suggestion
        return Result.conclude(piecesState);
```
piecesState에서 처리해야될 이유가 없어 보이네요

Reply:
그렇네요..! 다음 미션 때 반영하겠습니다